index.html

```diff
@@ -154,5 +154,31 @@ <h3>About me</h3>
             </div>
         </div>
     </section>
+    <section class="contact">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/{form_id}" method="post">
+            <input type="text" id="fullName" name="fullName" placeholder="Full Name" required>
+            <input type="text" id="firstName" name="firstName" placeholder="First Name" required>
+            <input type="text" id="lastName" id="lastName" placeholder="Last Name" required>
```

Comment:
Nice work adding `required` attr in the input field. But according to the requirement, you still need to add `maxlength="30"` attr in the name input field.
This is the requirement: [Name field accepts up to 30 chars](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#general-requirements).